prac_10/wiki.py

```diff
@@ -0,0 +1,20 @@
+"""
+CP1404/CP5632 Practical
```

Comment:
Overall, good work. I cannot fault most of the code here, only the one small issue that I have written about later. Good work! :D